RELEASE_NOTES.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+## 3.0.0 (2016-02-07)
```

Comment:
@charlierudolph didn't catch it earlier :( but wrong year here and in the manfiles